server/api/router.go

```diff
@@ -86,6 +86,8 @@ func createRouter(ctx context.Context, prefix string, svr *server.Server) *mux.R
 	apiRouter.HandleFunc("/config/label-property", confHandler.SetLabelProperty).Methods("POST")
 	apiRouter.HandleFunc("/config/cluster-version", confHandler.GetClusterVersion).Methods("GET")
 	apiRouter.HandleFunc("/config/cluster-version", confHandler.SetClusterVersion).Methods("POST")
+	apiRouter.HandleFunc("/config/replication-mode", confHandler.GetReplicationMode).Methods("GET")
+	apiRouter.HandleFunc("/config/replication-mode", confHandler.SetReplicationMode).Methods("POST")
```

Comment:
Do we also need to support changing it through /config API?

Reply:
I guess it is ok to not support. Just like `cluster-version`, `label-property`, etc.

Reply:
Does this config need SQL support? If it needs, I think it's better to support changing through /config. For `cluster-version` and `label-property`, PR #2354 is trying to support it.